locales/bn/json.json

```diff
@@ -212,15 +212,15 @@
     "Edit Attached": "সংযুক্ত সম্পাদনা করুন",
     "Edit Profile": "প্রোফাইল সম্পাদনা",
     "Editor": "সম্পাদক",
-    "Editor users have the ability to read, create, and update.": "সম্পাদক ব্যবহারকারীদের পড়তে তৈরি, এবং আপডেট করার ক্ষমতা রাখে ।",
+    "Editor users have the ability to read, create, and update.": "সম্পাদক ব্যবহারকারীরা পড়ার, তৈরির, এবং আপডেট করার ক্ষমতা রাখে।",
```

Comment:
"Editor users have the ability to read, create, and update.": "সম্পাদক ব্যবহারকারীরা পড়ার, তৈরির, এবং আপডেট করার ক্ষমতা রাখে।",
Here "আপডেট" can be replaced with "সম্পাদনা"

Reply:
should i make another pull request for this?